recipes/git-lfs/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "1.5.2" %}
+{% set download_url = "https://github.com/git-lfs/git-lfs/releases/download" %}
+
+package:
+  name: git-lfs
+  version: {{ version }}
+
+source:
+  url: {{ download_url }}/v{{ version }}/git-lfs-linux-amd64-{{ version }}.tar.gz  # [linux]
+  fn: git-lfs-linux-amd64-{{ version }}.tar.gz  # [linux]
+  sha256: 022a14ecafe4da0f637551c9971bdaabc1358395ede3611571cbe863e02a78c7  # [linux]
+
+  url: {{ download_url }}/v{{ version }}/git-lfs-darwin-amd64-{{ version }}.tar.gz  # [osx]
+  fn: git-lfs-darwin-amd64-{{ version }}.tar.gz  # [osx]
+  sha256: 2820b351b5ed44620ccfe090bd2f78f95c365875c3e64aeb271f7fe523d2e0c1  # [osx]
+
+  url: {{ download_url }}/v{{ version }}/git-lfs-windows-amd64-{{ version }}.zip  # [win]
+  fn: git-lfs-windows-amd64-{{ version }}.zip  # [win]
+  sha256: 8b4040e2021da421ae9c39d574e6826a5baa27a557e380ffacb63332bc436057  # [win]
+
+build:
+  number: 0
+
+test:
+  commands:
+    - git-lfs --help
+
+about:
+  home: https://git-lfs.github.com/
+  license: MIT
```

Comment:
You don't need to patch, you just need to point to the recipe dir, which conda-build makes available to you.